.readthedocs.yml

```
@@ -0,0 +1,14 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+    nodejs: "16"
```

Node.js 16 is EOL, use 18 or 20 instead?

https://endoflife.date/nodejs

```suggestion
    nodejs: "20"
```

---

We're still using NodeJS 16 in the build for the theme -- working to move to a newer NodeJS across the board in a follow up, right now.

---

https://github.com/pradyunsg/lutra/pull/58 is that PR.